planformcreator2/model/wing_exports.py

```diff
@@ -986,7 +986,63 @@ def write (self, aStream):
             self._write (aStream, "SPLITTERPOS=%s"             % "1400")
             self._write (aStream, self.end_tag)
 
+class Exporter_Csv(Exporter_Abstract):
+    """ 
+
+    Handle export of the planform to csv 
+
+    """
+
+    EXPORT_DIR_SUFFIX = "_csv"
+
+    def __init__(self, wing : Wing, dataDict: dict = None):
+        super().__init__(wing, dataDict=dataDict)
+ 
+
+    def _as_dict (self) -> dict:
+        """ returns a data dict with the paramters of self"""
+
+        d = {}
+        toDict (d, "export_dir",        self._export_dir) 
+        d.update(self.exporter_airfoils._as_dict())
+        return d
+
+
+    @property
+    def csv_filename(self): 
+        return self._wing.parm_fileName_stem + '_wing.csv'
+
+
+    def do_it (self): 
+        """ 
+        main entry: start the export to the file defined in self parameters.
+        Returns a message string what was done 
+        """
+
+        self._ensure_export_dir()
+
+        # plot the different parts in a dxf document 
+
+        csv_pathFileName = os.path.join (self.export_dir_abs, self.csv_filename)
+
+        with open (csv_pathFileName, mode='w', newline='') as csvfile:
+            csv_writer = csv.writer (csvfile, delimiter=';', quotechar='"', quoting=csv.QUOTE_MINIMAL)
+
+            # write header 
+            csv_writer.writerow (["xn", "x", "c", "le_y", "te_y", "twist_deg", "flap_cn", "flap_group", "airfoil_file"])
```

Comment:
Twist gibt es leider nicht ...
```suggestion
            csv_writer.writerow (["xn", "x", "c", "le_y", "te_y", "flap_cn", "flap_group", "airfoil_file"])
```